### PR TITLE
DO NOT MERGE: SNS deployment principal - proposal text

### PR DESCRIPTION
**Pull request by bitdivine**
- [x] Approve deployment text
- [x] Ensure that team members have access to the key
- [x] Make proposal
- [x] Deploy one canister to the SNS subnet.  Important; this MUST be the first deployment by this principal.

### CODE REVIEW

**Review by dubovitskaya — COMMENTED** [review]
Thanks a lot Max, looks good! I would only add that as soon as the Aggregator canister is deployed the principal will be removed from the whitelist.

**Comment by bitdivine** [comment]
@dubovitskaya - I have added text to that effect.

File: sns_whitelist_command.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+: Check that the principal is correct
+[[ "$(dfx identity get-principal --identity sns-subnet)" == "wgfa2-l65ry-bjl6l-idehu-56rsz-srpsc-km5fe-jgglp-p5up7-npbep-5ae" ]] || {
```
Comment (frederikrothenberger) [inline sns_whitelist_command.sh:5]: I can confirm that the principal of the `sns-subnet` identity that I received from @bitdivine does have the principal `wgfa2-l65ry-bjl6l-idehu-56rsz-srpsc-km5fe-jgglp-p5up7-npbep-5ae`.

File: sns_whitelist_command.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+cargo run --bin ic-admin -- --nns-url https://nns.ic0.app/ \
+	--use-hsm \
+	--pin $(cat ~/.hsm-pin) \
+	--key-id 01 \
+	--slot 0 \
+	propose-to-set-authorized-subnetworks \
+	--proposer "$NEURON" \
+        --summary-file sns_principal.md \
+	--subnets x33ed-h457x-bsgyx-oqxqf-6pzwv-wkhzr-rm2j3-npodi-purzm-n66cg-gae \
```
Comment (frederikrothenberger) [inline sns_whitelist_command.sh:21]: According to the dashboard, this is indeed the SNS application subnet.

File: sns_whitelist_command.sh (hunk 1 above)
Comment (lmuntaner) [reply sns_whitelist_command.sh:5]: I pulled the @bitdivine 's key, how do I get the principal?

Comment (frederikrothenberger) [reply sns_whitelist_command.sh:5]: Run the following commands:
* `dfx identity use sns-subnet`
* `dfx identity --network ic get-principal`

Comment (lmuntaner) [reply sns_whitelist_command.sh:5]: I need to add it to dfx first, right?

Did `gpg` create a pem file?

Comment (bitdivine) [reply sns_whitelist_command.sh:5]: I think these are slightly crossed - @lmuntaner , I forget,have you published an encryption key?  I think the last time I looked there was an RSA key for singing only, not encryption.  So I could not encrypt a file to send to you.

Pulling my key lets you send me messages, no the other way around.  You could send me a secret key and I encrypt the dfx identity data under that secret key then send you the encryped file.

Comment (lmuntaner) [reply sns_whitelist_command.sh:5]: aaahhh, ok, I'll do that then.

**Comment by bitdivine** [comment]
Made a canister and confirmed that it is on the sns subnet:
```
$ dfx ledger --network ic create-canister $(dfx identity get-principal) --icp 4
Transfer sent at block height 5511581
Canister created with id: "3r4gx-wqaaa-aaaaq-aaaia-cai"
$ dfx identity whoami
sns-subnet
```

**Comment by bitdivine** [comment]
Made a wallet canister:
```
$ dfx ledger --network ic create-canister $(dfx identity get-principal) --icp 2
Transfer sent at block height 5532941
Canister created with id: "3w5ad-3iaaa-aaaaq-aaaiq-cai"
```

**Comment by bitdivine** [comment]
```
aggregator-params (2:42)$ dfx deploy sns_aggregator --network ic
```

**Pull request closed without merging**